syntax = "proto3";

import "empty.proto";

package kaonic;

service Device {
    rpc GetInfo(kaonic.Empty) returns (kaonic.Empty) {}
    rpc GetStatistics(kaonic.Empty) returns (kaonic.Empty) {}
}
